pcdsdevices/sqr1.py

```diff
@@ -0,0 +1,141 @@
+"""
+    Ophyd class for square-one tri-sphere motion system
+    (https://www.sqr-1.com/tsp.html)
+"""
+
+import logging
+import typing
+
+from ophyd.device import Device
+from ophyd.device import FormattedComponent as FCpt
+from ophyd.pv_positioner import PVPositionerIsClose
+from ophyd.signal import EpicsSignal
+from ophyd.status import AndStatus
+from ophyd.status import wait as status_wait
+
+logger = logging.getLogger(__name__)
+
+
+class SQR1Axis(PVPositionerIsClose):
+    setpoint = FCpt(EpicsSignal, '{prefix}:TARGET:{_axis}', kind='normal')
+    readback = FCpt(EpicsSignal, '{prefix}:TARGET:{_axis}:RBV', kind='hinted')
+    actuate = FCpt(EpicsSignal, '{prefix}:MOV', kind='normal')
+    actuate_value = 1
+    stop_signal = FCpt(EpicsSignal, '{prefix}:KILL', kind='normal')
+    stop_value = 1
+
+    def __init__(self,
+                 prefix,
+                 axis: str,
+                 sync_setpoints: typing.function = None,
+                 **kwargs
+                 ):
+        self.prefix = prefix
```

Comment:
Optional suggestion: This is ok but is actually redundant because it is set during `super().__init__`, you could remove it

Reply:
you are right, I deleted it